cime_config/buildnml

```diff
@@ -203,15 +203,16 @@ def buildnml(case, caseroot, compname):
         clm_usrdat_name = case.get_value("CLM_USRDAT_NAME")
         clmusr = " -clm_usr_name %s " % clm_usrdat_name
         # Write warning about initial condition data
-        if "NEON" in clm_usrdat_name and clm_force_coldstart == "off":
-            if ("_transient" in clm_nml_use_case) and (
-                re.fullmatch(r"\w\w\w\w\.transient", casename) is None
-                or clm_usrdat_name is "NEON.PRISM"
-            ):
-                logger.warning(
-                    "WARNING: Do you have appropriate initial conditions for this simulation?"
-                    + " Check that the finidat file used in the lnd_in namelist is appropriately spunup for your case"
-                )
+        if ("NEON" in clm_usrdat_name) or ("PLUMBER" in clm_usrdat_name):
+            if clm_force_coldstart == "off":
+                if ("_transient" in clm_nml_use_case) and (
+                    re.fullmatch(r"\w\w\w\w\.transient", casename) is None
```

Comment:
@TeaganKing and I realized from this string check that PLUMBER should be in a separate if-block with a different string check for it to work correctly for PLUMBER.

@TeaganKing and I also discussed the order that these three related PRs will get merged:
https://github.com/ESCOMP/CDEPS/pull/262
This PR
#2406 

I will preemptively approve this PR.

Reply:
Thanks Sam! I restructured that bit, but am running into some more issues with black and will get back to this hopefully in a few hours!

Reply:
Ok, I got back early from lunch and fixed this! This file is now passing black checks and includes an updated PLUMBER-specific `fullmatch`.